### DISCUSSION TITLE
How do I update the impact categories?

### DISCUSSION THREAD

**Discussion by JuliSant** [LCA-ActivityBrowser/activity-browser#983] [Q&A]
Hello

for some reason, the ReCiPe Endpoint categories are given in points in my AB, while I have colleagues that have DALYs, species/y, and dollars, as it should be. How can I update this impact categories?

Thanks in advance!

Kind regards
Julia

![Picture2](https://github.com/LCA-ActivityBrowser/activity-browser/assets/127324445/f34f498f-c507-4502-a106-51837828ebd7)

**Comment by Zoophobus** [comment]
Hi Julia, could you try and find the version of the AB that you have? This should be in the about section of the help menu, another factor that might be involved in this is the version of brightway2 packages that you have. For this running `conda list bw` and `conda list brightway` should provide enough information. 

KR,
Jon

**Comment by JuliSant** [comment]
Dear Jonathan

Thanks for your answer! I have the version 2.6.9 of the AB.

```
(ab) C:\>conda list bw
# packages in environment at C:\Users\Jsantolin\Miniconda3\envs\ab:
#
# Name                    Version                   Build  Channel
bw2analyzer               0.10                       py_1    cmutel
bw2calc                   1.8.1                      py_2    cmutel
bw2data                   3.6.5                      py_0    cmutel
bw2io                     0.8.6                      py_1    cmutel
bw2parameters             0.7                pyhd8ed1ab_0    conda-forge
bw_migrations             0.1                        py_0    cmutel
libwebp                   1.2.4                h8ffe710_0    conda-forge
libwebp-base              1.2.4                h8ffe710_0    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge

(ab) C:\>conda list brightway
# packages in environment at C:\Users\Jsantolin\Miniconda3\envs\ab:
#
# Name                    Version                   Build  Channel
brightway2                2.4.2                      py_5    cmutel
```


Do you know how I can fix it?


Best
Julia

**Reply by Zoophobus** [reply]
Hi Julia,

others will probably know better than me, but the first place I'd start is by updating the bw2io version. This is where the environmental flows are managed and the code for updating missing CFs, you could try upgrading this to version 0.8.7. I would also recommend you to update the AB too, there are several changes (hopefully improvements) in the AB since you last installed it.

KR
Jon

**Reply by JuliSant** [reply]
Hi Jon,

so I updated the AB and then downgraded to the 0.8.7 version of bw2io because I am using ecoinvent 3.8 (following this: https://github.com/brightway-lca/brightway2-io). I still see points..

Any other ideas?

Best
Julia

**Comment by marc-vdm** [comment]
Hi Julia, is there a specific reason you want to use `ReCiPe Midpoint (H, A)`? The ReCiPe v1.13 categories that are available in AB do have the units you're looking for. For me the `ReCiPe Midpoint (H, A)` also shows points, but AFAIK v1.13 is newer, though don't quote me on that.

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/34626062/eb945bd9-a581-42a4-8a81-b10bf8a91325)

**Comment by JuliSant** [comment]
Hi Marc, actually, I want to use endpoints, that's the thing. The midpoints also have the correct units for me.

Best
Julia

**Comment by marc-vdm** [comment]
I see, my bad. I think the best option is to make an issue over at [Brightway-2](https://github.com/brightway-lca/brightway2) and check with them. We get our impact categories from them, so we can't really make changes anyway.

Also, are you certain that the same impact categories with your colleagues give them the correct units? If so, could you also ask them to give us the info for the AB version, `conda list bw` and `conda list brightway`? That will be interesting for us to know.

**Comment by JuliSant** [comment]
Thanks Marc, I have done that now! My colleague uses the 2.6.9 version of the AB.

(base) C:\Users\hanso>conda list bw
packages in environment at C:\Users\hanso\anaconda3:

 Name                    Version                   Build  Channel
libwebp                   1.2.2                h2bbff1b_0
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge

 

(base) C:\Users\hanso>conda list brightway
packages in environment at C:\Users\hanso\anaconda3:

 Name                    Version                   Build  Channel
brightway2-conda-installer 0.0.4                      py_0    haasad

**Comment by marc-vdm** [comment]
Oops sorry, I should have added this, could they first do

`conda activate ab` or perhaps `ab_dev` (whatever they used as a name for the environment) and then do the above again?

**Reply by JuliSant** [reply]
True, I forgot to tell him! So:

(ab) C:\Users\hanso>conda list bw
 packages in environment at C:\Users\hanso\anaconda3\envs\ab:

 Name                    Version                   Build  Channel
bw2analyzer               0.10                       py_1    cmutel
bw2calc                   1.8.1                      py_2    cmutel
bw2data                   3.6.5                      py_0    cmutel
bw2io                     0.8.8                      py_0    cmutel
bw2parameters             0.7                pyhd8ed1ab_0    conda-forge
bw_migrations             0.1                        py_0    cmutel
libwebp                   1.2.4                h8ffe710_0    conda-forge
libwebp-base              1.2.4                h8ffe710_0    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge

 

 

(ab) C:\Users\hanso>conda list brightway
 packages in environment at C:\Users\hanso\anaconda3\envs\ab:

 Name                    Version                   Build  Channel
brightway2                2.4.2                      py_6    cmutel


has context menu

**Comment by marc-vdm** [comment]
@Zoophobus Seems to me somehow 2.6.9 installed with `bw2io 0.8.8` for the other user.

@JuliSant Your colleague has a version of  `bw2io` (0.8.8) that's actually supposed to be used only with ecoinvent 3.9 and newer, while your version (0.8.7) is supposed to be used only with ecoinvent 3.8 and older. Apparently, this change to the units was made between these two versions. While there is no guarantee things will go wrong when you mix these, there's a good chance it will. There's unfortunately no `bw2io` version that works for all ecoinvent version yet.

Depending on which version of ecoinvent you -and your colleague too- are using, we'd recommend to get the matching `bw2io` version. [More information on how to do that here](https://github.com/LCA-ActivityBrowser/activity-browser#install-the-ab-with-ecoinvent-39). Of course, we're happy to help you figure it out if anything is unclear!

**Reply by JuliSant** [reply]
Yes, I updated everything but then downgraded to 0.8.7 because I'm working with ecoinvent 3.8. What do the points mean though? Is there a normalization? Or does it only say points but it means something else (e.g. dalys)?

**Reply by marc-vdm** [reply]
@cmutel Could you help @JuliSant and me understand what the points mean in the ReCiPe endpoint categories from `bw2io 0.8.7`? It's different from `bw2io 0.8.8` where there are units.
